jena-text/src/main/java/org/apache/jena/query/text/assembler/TextAssembler.java

```diff
@@ -29,14 +29,15 @@ public static void init()
         AssemblerUtils.registerDataset(TextVocab.textDataset,      new TextDatasetAssembler()) ;
         
         Assembler.general.implementWith(TextVocab.entityMap,        new EntityDefinitionAssembler()) ;
-        Assembler.general.implementWith(TextVocab.textIndexSolr,    new TextIndexSolrAssembler()) ; 
         Assembler.general.implementWith(TextVocab.textIndexLucene,  new TextIndexLuceneAssembler()) ;
         Assembler.general.implementWith(TextVocab.standardAnalyzer, new StandardAnalyzerAssembler()) ;
         Assembler.general.implementWith(TextVocab.simpleAnalyzer,   new SimpleAnalyzerAssembler()) ;
         Assembler.general.implementWith(TextVocab.keywordAnalyzer,  new KeywordAnalyzerAssembler()) ;
         Assembler.general.implementWith(TextVocab.lowerCaseKeywordAnalyzer, new LowerCaseKeywordAnalyzerAssembler()) ;
         Assembler.general.implementWith(TextVocab.localizedAnalyzer, new LocalizedAnalyzerAssembler()) ;
         Assembler.general.implementWith(TextVocab.configurableAnalyzer, new ConfigurableAnalyzerAssembler()) ;
+        Assembler.general.implementWith(TextVocab.textIndexES,  new TextIndexESAssembler()) ;
```

Comment:
minor style issue, but please move this next to TextIndexLuceneAssembler a few lines up, because they're similar